### PR TITLE
refactor: accessibility and responsive

### PR DESCRIPTION
**Pull request by opakholis**
hi @nyancodeid, i just changed behaviour button of **choose language** to `cursor: pointer`, i think you forgot about this. 
and also made padding better on smaller device.

### CODE REVIEW

**Comment by opakholis** [comment]
here i attach my screen

https://user-images.githubusercontent.com/49778014/129462523-a54d43f1-fc6b-4a2c-bcf4-38a9bda69fac.mp4

File: src/components/ColorPicker/ColorResult.vue [hunk 1]
```diff
@@ -131,4 +131,10 @@ export default {
     }
   }
 }
+
+@media only screen and (max-width: 640px) {
```
Comment (nyancodeid) [inline src/components/ColorPicker/ColorResult.vue]: Hi @opxop , can you make max-width to `(max-width: 768px)` like i did.

**Review by nyancodeid — CHANGES_REQUESTED** [review]
Hi thanks for PR. I appreciate and it's look good now.

File: src/components/ColorPicker/ColorResult.vue (hunk 1 above)
Comment (opakholis) [reply src/components/ColorPicker/ColorResult.vue]: I've made changes, thanks for correcting me. i think it's better now

Comment (nyancodeid) [reply src/components/ColorPicker/ColorResult.vue]: Thank you

File: src/components/ColorPicker/ColorResult.vue [hunk 2]
```diff
@@ -87,15 +87,13 @@ export default {
...
+      margin: 8px 8px 16px;
       color: var(--darker-color);
       font-size: 14px;
     }
 
     .video-color--result-item {
-      margin-bottom: 12px;
-      margin-right: 8px;
-      margin-left: 8px;
+      margin: 0 8px 12px;
```
Comment (opakholis) [inline src/components/ColorPicker/ColorResult.vue:96]: compared to this:
```css
margin-bottom: 12px;
margin-right: 8px;
margin-left: 8px;
```
can be shorthand by writing `margin: 0 8px 12px;`

reference: https://web.dev/learn/css/spacing/#margin

**Review by nyancodeid — APPROVED** [review]
LGTM!

File: src/components/ColorPicker/ColorResult.vue (hunk 2 above)
Comment (nyancodeid) [reply src/components/ColorPicker/ColorResult.vue:96]: Nice catch, thanks

**Comment by nyancodeid** [comment]
@all-contributors please add @opxop for code, design

**Pull request merged**